nx.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
Do we want add tags, and impose constraints? https://nx.dev/structure/monorepo-tags
Currently it seems I am allowed to import from feature X in feature Y.